src/main/java/teammates/common/util/Utils.java

```diff
@@ -35,5 +37,23 @@ public static Gson getTeammatesGson() {
                 .disableHtmlEscaping()
                 .create();
     }
+    
+    /**
+     * Convert a list of {@link FeedbackParticipantType} to a string where each element will be separated by
+     * a delimiter.
+     */
+    public static String joinParticipantTypes(List<FeedbackParticipantType> participants, String delimiter) {
```

Comment:
Should this method have to be limited to `FeedbackParticipantType` or can it be used on any List? The latter is preferred (utility methods should be general and reusable)